src/Compilers/VisualBasic/Portable/Compilation/SyntaxTreeSemanticModel.vb

```diff
@@ -864,7 +864,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' </summary>
         Private Function CheckSymbolLocationsAgainstSyntax(symbol As NamedTypeSymbol, nodeToCheck As VisualBasicSyntaxNode) As NamedTypeSymbol
             For Each location In symbol.Locations
-                If location.SourceTree Is Me.SyntaxTree AndAlso nodeToCheck.Span.Contains(location.SourceSpan.Start) Then
+                If location.SourceTree Is Me.SyntaxTree AndAlso nodeToCheck.Span.Contains(location.SourceSpan) Then
```

Comment:
> Without this, the VB implementation would return `Nothing` for types if the user hasn't typed a name yet.

I am probably missing something obvious, but it looks like the condition became tighter. So, I am not sure how it makes the function to return Nothing less often. I think, first a problem should be demonstrated and only then addressed. Without that, I wouldn't keep the change.     

---
In reply to: [67624193](https://github.com/dotnet/roslyn/pull/12092#discussion_r67624193) [](ancestors = 67624193)